packages/discovery-react-components/src/components/StructuredQuery/components/RuleRow/RuleRow.tsx

```diff
@@ -45,16 +53,26 @@ export const RuleRow: FC<RuleRowProps> = ({
   const showRemoveRuleRowButton =
     structuredQuerySelection.groups[groupId].rows.length > 1 || !isTopLevelGroup;
 
+  // TODO Check out missing prop warning on <ComboBox />
   return (
     <div className={structuredQueryRulesClass} data-testid={`rule-row-${groupId}`}>
-      <ComboBox
-        id={`structured-query-rules-field-${groupId}`}
-        // TODO: Items is empty for now as it's a required field and retrieving fields for the dropdown
-        // and adding them as items will be addressed in a future issue
-        items={[]}
-        placeholder={messages.fieldDropdownPlaceholderText}
-        titleText={messages.fieldDropdownTitleText}
-      />
+      {isLoading || !projectFields ? (
```

Comment:
could we inline the loading message in the `placeholder` attribute then have `projectFields` default to an array instead of this ternary?